src/System.Collections.NonGeneric/tests/System.Collections.NonGeneric.Tests.csproj

```diff
@@ -147,14 +147,17 @@
     <Compile Include="$(CommonTestPath)\System\Diagnostics\DebuggerAttributes.cs">
       <Link>Common\System\Diagnostics\DebuggerAttributes.cs</Link>
     </Compile>
-  </ItemGroup>
-  <!-- Performance Tests -->
-  <ItemGroup Condition="'$(RunPerfTestsForProject)' == 'true'">
+    <!-- Performance Tests -->
     <Compile Include="Performance\Perf.HashTable.cs" />
     <Compile Include="$(CommonTestPath)\System\PerfUtils.cs">
       <Link>Common\System\PerfUtils.cs</Link>
     </Compile>
   </ItemGroup>
+  <!-- Optimizations to configure Xunit for performance -->
+  <ItemGroup Condition="'$(Performance)' == 'true'">
```

Comment:
I would prefer that as well, but it would require that every project have a reference to Microsoft.Xunit.Performance. It would also make every test project be configured with the OptimizeForBenchmarks assembly attribute which may or may not be troublesome.